Include/internal/pycore_code.h

```diff
@@ -24,9 +27,117 @@ struct _PyOpcache {
     char optimized;
 };
 
+
+// XXX Can we do this with an enum instead?
+typedef unsigned char _PyFastLocalKind;
+#define CO_FAST_POSONLY     0x01
+#define CO_FAST_POSORKW     0x02
+#define CO_FAST_VARARGS     0x04
+#define CO_FAST_KWONLY      0x08
+#define CO_FAST_VARKWARGS   0x10
+#define CO_FAST_LOCALONLY   0x20
+#define CO_FAST_CELL        0x40
+#define CO_FAST_FREE        0x80
+#define CO_FAST_LOCAL (CO_FAST_POSONLY | CO_FAST_POSORKW | CO_FAST_VARARGS | \
+                       CO_FAST_KWONLY | CO_FAST_VARKWARGS |                  \
+                       CO_FAST_LOCALONLY)
```

Comment:
Sorry if this is naive, but can't you condense this into two lines?
```suggestion
#define CO_FAST_LOCAL (CO_FAST_POSONLY | CO_FAST_POSORKW | CO_FAST_VARARGS | \
                       CO_FAST_KWONLY | CO_FAST_VARKWARGS | CO_FAST_LOCALONLY)
```

Reply:
Your question isn't naive.  The reason I did what I did isn't obvious.  My preference is to group similar items in code, usually by line.  In this case, the first five items are all kinds of arguments, while the last one isn't.  I would have put the first group all on one line if that line weren't so long.

I'll clarify this, one way or another.  Thanks for pointing it out.